src/Abstractions/TaskOptions.cs

```diff
@@ -28,7 +25,10 @@ internal TaskOptions(Builder builder)
     /// </summary>
     public CancellationToken CancellationToken { get; }
 
-    internal AsyncRetryHandler? RetryHandler { get; }
+    /// <summary>
+    /// Gets the retry handler.
+    /// </summary>
+    public AsyncRetryHandler? RetryHandler { get; }
```

Comment:
Why did this need to be made public?

Reply:
It is used in the worker project. Given `AsyncRetryHandler` is a public delegate, I am not concerned with this being public. But if you have concerns, I will make a note to re-evaluate this later and see if there is a better way to expose this publicly.